packages/consent-manager/index.js

```diff
@@ -80,7 +82,7 @@ export default class ConsentManager extends Component {
 
   render() {
     return (
-      <div id="g-consent-manager">
+      <div className={classNames(s.root, this.props.className)}>
```

Comment:
Added support for `className` to ensure consumers can override hiding from Percy if needed.